Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM ubuntu
+
+# install golang
+
+RUN apt-get update && \
+    apt-get install -y curl && apt-get install -q -y \
+    gnupg2 \
+    git \
+    vim \
+    openssh-client \
+    curl \
+    wget  \
+    && rm -rf /var/lib/apt/lists/*
+
+ENV GOLANG_VERSION 1.14.3
+
+RUN curl -sSL https://storage.googleapis.com/golang/go$GOLANG_VERSION.linux-amd64.tar.gz \
+		| tar -v -C /usr/local -xz
+
+ENV PATH /usr/local/go/bin:$PATH
+RUN mkdir -p /go/src /go/bin && chmod -R 777 /go
+ENV GOROOT /usr/local/go
+ENV GOPATH /go
+ENV PATH /go/bin:$PATH
```

Comment:
We could use `golang:1.14` image here instead :angel:

Reply:
yeah, Need to optimise this Dockerfile, I will update that with base image `golang:1.14` and push changes

Reply:
Updated :+1: